assets/wizards/engagement/views/reader-activation/index.js

```diff
@@ -112,17 +103,53 @@ export default withWizardScreen( () => {
 						onChange={ value => updateConfig( 'terms_url', value ) }
 					/>
 				</Grid>
-			</Card>
-			{ isActiveCampaign && (
-				<ActiveCampaign
-					value={ { masterList: config.active_campaign_master_list } }
-					onChange={ ( key, value ) => {
-						if ( key === 'masterList' ) {
-							updateConfig( 'active_campaign_master_list', value );
-						}
-					} }
+				<hr />
+				<SectionHeader
+					title={ __( 'Email Service Provider and Newsletters Integration', 'newspack' ) }
```

Comment:
Non-blocking: this seems a bit wordy and redundant with the description below. How about:

```suggestion
					title={ __( 'Email Service Provider Settings', 'newspack' ) }
```